content/geometry/object-human/human-structure.adoc

```diff
@@ -7,6 +7,10 @@
 legend
 |_ [[../geometry/object-human/human-index.html#_grp_root Grp_Root]]
   |_ [[../geometry/object-human/human-index.html#_armature_name Armature_<Name>]]
+    |_ [[../geometry/object-human/human-index.html#_accessories_name Accessories_<Name>]]
+    |_ [[../geometry/object-human/human-index.html#_body_name Body_<Name>]]
+    |_ [[../geometry/object-human/human-index.html#_clothing_name Clothing_<Name>]]
+    |_ [[../geometry/object-human/human-index.html#_hair_name Hair_<Name>]]
```

Comment:
I would like to keep these nodes below the bones as this is how it is displayed in 3D modeling tools (at least in Blender).

Reply:
Moved the four chapters below the bones (in node structure and in the chapter itself)